inlong-dashboard/src/metas/stream/index.tsx

```diff
@@ -142,27 +142,27 @@ const fieldsDefault: FieldItemType[] = [
       options: [
         {
           label: i18n.t('meta.Stream.DataSeparator.Space'),
-          value: '32',
+          value: ' ',
```

Comment:
Spaces are not easy to identify after being saved in the back-end database;

In addition, if the user needs to set a special character, there may be problems with the front-end display and back-end storage.

Therefore, it is recommended to use ASCII code to save.